base/shell/explorer/explorer.cpp

```diff
@@ -87,6 +87,45 @@ HideMinimizedWindows(IN BOOL bHide)
 }
 #endif
 
+static BOOL
+IsExplorerSystemShell()
+{
+    HKEY hKeyWinlogon;
+	WCHAR szShell[256];
+	WCHAR szPath[MAX_PATH];
```

Comment:
Fixing indentation (tab replaced by 4 spaces):
```suggestion
    WCHAR szPath[MAX_PATH];
```

Reply:
Fixed in last commit